base/testfiles-lthooks/lthooks-001.tlg

```diff
@@ -109,6 +109,24 @@ All initialized (non-empty) hooks:
  file/after/ulem.sty -> \__hook_toplevel file/after/ulem.sty \def \@hspace ##1{\begingroup \setlength \skip@ {##1}\edef \x {\endgroup \hskip \the \skip@ \relax }\x }\__hook_next file/after/ulem.sty  
  file/after/varwidth.sty -> \__hook_toplevel file/after/varwidth.sty \FirstAidNeededT {varwidth}{sty}{....-..-.. ver 0.92; \space Variable-width minipages}{\def \@vwid@sift {\skip@ \lastskip \unskip \ifdim \lastskip =\z@ \unskip \fi \dimen@ \lastkern \unkern \count@ \lastpenalty \unpenalty \setbox \z@ \lastbox \ifvoid \z@ \advance \sift@deathcycles \@ne \else \sift@deathcycles \z@ \fi \ifnum \sift@deathcycles >33 \let \@vwid@sift \relax \PackageWarning {varwidth}{Failed to reprocess entire contents}\fi \ifnum \count@ =\@vwid@preeqp \@vwid@eqmodefalse \fi \ifnum \count@ =\@vwid@posteqp \@vwid@eqmodetrue \fi \ifnum \count@ =\@vwid@toppen \let \@vwid@sift \relax \else \ifnum \count@ =\@vwid@offsets \@vwid@setoffsets \else \ifnum \count@ =\@vwid@postw \else \@vwid@resetb \fi \@vwid@append \fi \fi \@vwid@sift }}\__hook_next file/after/varwidth.sty  
  xxx -> foobar\__hook_toplevel xxx \__hook_next xxx  
+! LaTeX hooks Error: Cannot remove chunk 'label3' from hook 'xxx' because it
+(hooks)              does not exist in that hook.
+Type <return> to continue.
+ ...                                              
+l. ...\hook_gremove_code:nn{xxx}{label3}
+                                        % errors because label3 doesn't exist
+LaTeX does not know anything more about this error, sorry.
+Try typing <return> to proceed.
+If that doesn't work, type X <return> to quit.
```

Comment:
if we keep it an error then there should also be a help text, but as I said I think warning is probably sufficient